apps/ima/index.js

```diff
@@ -33,9 +33,40 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/exception', // TO BE UPDATED AS STEPS ARE ADDED
       backLink: 'current-progress'
     },
+    '/exception': {
+      behaviours: SaveFormSession,
+      fields: [
+        'does-exception-apply',
+        'does-exception-apply-detail'
+      ],
+      template: 'does-exception-apply',
+      locals: { showSaveAndExit: true },
+      next: '/threatened-life-or-liberty' // TO BE UPDATED AS STEPS ARE ADDED
```

Comment:
I don't think the comment is needed here as '/threatened-life-or-liberty'  is the correct next step for this step

Reply:
According to the Figma design, the next step would be the 'removal-condition' screen (IMB-53)